client/my-sites/hosting/hosting-activate.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import MainComponent from 'components/main';
+import HeaderCake from 'components/header-cake';
+import EligibilityWarnings from 'blocks/eligibility-warnings';
+import PageViewTracker from 'lib/analytics/page-view-tracker';
+import { initiateThemeTransfer } from 'state/themes/actions';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+
+const HostingActivate = ( { initiateTransfer, siteId, siteSlug, translate } ) => {
+	const getBackUrl = () => {
+		return `/hosting-config/${ siteSlug }`;
+	};
+
+	const transferInitiate = () => {
+		initiateTransfer( siteId, null, null );
+		page( getBackUrl() );
+	};
+
+	return (
+		<MainComponent>
```

Comment:
I don't have strong feelings for one view vs two, but I'd recommend displaying cards with the disabled state here:

```jsx
<SFTPCard disabled /> 
<PhpMyAdminCard disabled />
< PhpVersionCard disabled />
```

OR move the Notice+Eligibility Warnings to the main view. This is pretty barren visually without the disabled cards as a hint of what this page does.